Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,31 @@
-import React from 'react';
-import './App.css';
-
+import React from "react";
+import "./App.css";
+import Display from "./components/DisplayComponents/CalculatorDisplay";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import ActionButton from "./components/ButtonComponents/ActionButton";
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <section>
+      <Display />
+      <div>
+        <ActionButton actionButtonStyle={"action"} text={"clear"} />
```

Comment:
```suggestion
        <ActionButton actionButtonStyle="action" text="clear" />
```
You only need the curly braces where you want a prop or attribute to be evaluated as JS: for example if you have a variable there, or an expression. Since you are just passing a string, the "" will be fine. Same for all the buttons